src/cmd/cli/command/compose.go

```diff
@@ -15,9 +16,30 @@ import (
 	"github.com/DefangLabs/defang/src/pkg/types"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
 	"github.com/bufbuild/connect-go"
+	compose "github.com/compose-spec/compose-go/v2/types"
 	"github.com/spf13/cobra"
 )
 
+var managed_resources = []string{"x-defang-redis", "x-defang-postgres"}
+
+func IsOnlyManagedResources(project *compose.Project) bool {
+	for _, service := range project.Services {
+		if len(service.Extensions) == 0 {
```

Comment:
For readability, consider extracting a separate `func IsManagedService`, which currently uses the list of services, but eventually could be replaced by a boolean from ServiceInfo, once we add one.